homeassistant/components/life360/device_tracker.py

```diff
@@ -168,12 +168,6 @@ def force_update(self) -> bool:
         """Return True if state updates should be forced."""
         return False
 
-    @property
-    def available(self) -> bool:
-        """Return if entity is available."""
-        # Guard against member not being in last update for some reason.
-        return super().available and self._data is not None
```

Comment:
This ain't right?

Reply:
The override is not needed anymore. Given the other change the implementation in the superclass is sufficient.